main.js

```diff
@@ -1,43 +1,42 @@
+const hamburguerContainer = document.querySelector('.header-menu-hamburguer');
 
-// 
-let generator = document.querySelector('.header-menu-hamburguer')
-// 
+const hamburguerIcon = document.querySelector('.header-menu-hamburguer img');
+const closeIcon = document.querySelector('.header-menu-hamburguer i');
 
-let target1 = document.querySelector('.header-menu-hamburguer img')
-let target2 = document.querySelector('.header-menu-hamburguer i')
+const menuIcon = document.querySelector('.menu-mobil');
 
-let modal = document.querySelector('.menu-mobil')
+const headerBlur = document.querySelector('.hero');
 
-let header = document.querySelector('.hero')
+const logoBlur = document.querySelector('.header > p');
 
-let logo = document.querySelector('.header > p')
-
-
-document.documentElement.addEventListener('click', e=>{
-  console.log(e.target)
-
-  if(e.target.matches('#uno')){
-    modal.classList.toggle('show')
-    header.classList.toggle('blur')
-   logo.classList.toggle('blur')
+document.documentElement.addEventListener('click', (e) => {
+  if (e.target.matches('#uno')) {
+    menuIcon.classList.toggle('show');
+    headerBlur.classList.toggle('blur');
+    logoBlur.classList.toggle('blur');
+    hamburguerIcon.classList.toggle('hide');
+    closeIcon.classList.toggle('show');
   }
-  if(e.target.matches('#dos')){
-    modal.classList.toggle('show')
-    header.classList.toggle('blur')
-   logo.classList.toggle('blur')
+  if (e.target.matches('#dos')) {
+    menuIcon.classList.toggle('show');
+    headerBlur.classList.toggle('blur');
+    logoBlur.classList.toggle('blur');
+    hamburguerIcon.classList.toggle('hide');
+    closeIcon.classList.toggle('show');
   }
-  if(e.target.matches('#tres')){
-    modal.classList.toggle('show')
-    header.classList.toggle('blur')
-   logo.classList.toggle('blur')
+  if (e.target.matches('#tres')) {
+    menuIcon.classList.toggle('show');
+    headerBlur.classList.toggle('blur');
+    logoBlur.classList.toggle('blur');
+    hamburguerIcon.classList.toggle('hide');
+    closeIcon.classList.toggle('show');
   }
-})
-
-
-generator.addEventListener('click', ()=>{ 
-   target1.classList.toggle('hide') 
-   target2.classList.toggle('show') 
-   modal.classList.toggle('show')
-   header.classList.toggle('blur')
-   logo.classList.toggle('blur')
-})
\ No newline at end of file
+});
+
+hamburguerContainer.addEventListener('click', () => {
+  hamburguerIcon.classList.toggle('hide');
+  closeIcon.classList.toggle('show');
+  menuIcon.classList.toggle('show');
+  headerBlur.classList.toggle('blur');
+  logoBlur.classList.toggle('blur');
+});
```

Comment:
- [ ] Good job so far 👍  But I noticed that your code is quite repetitive 😢 I would like to recommend you write a function that contains a specific logic and then call it multiple times in our code to make your code DRY . You can check the example below ⬇️ 

```
// Mobile Menu Variables
const containerMenu = document.querySelector('.menu');
const closeMenu = document.querySelector('.menu-close');
const openMenu = document.querySelector('.menu-open');
const linkMenu = document.querySelectorAll('.menu a');

// Mobile Menu Functions
function menuOpen() {
  containerMenu.style.display = 'flex';
}
function menuClose() {
  containerMenu.style.display = 'none';
}

// Mobile Menu Events
linkMenu.forEach((link) => {
  link.addEventListener('click', menuClose);
});

openMenu.addEventListener('click', menuOpen);
closeMenu.addEventListener('click', menuClose);
```